paddle/parameter/Argument.h

```diff
@@ -144,8 +144,8 @@ struct Argument {
   }
   size_t getFrameHeight() const { return frameHeight; }
   size_t getFrameWidth() const { return frameWidth; }
-  void setFrameHeight(size_t h) { frameHeight = h; }
-  void setFrameWidth(size_t w) { frameWidth = w; }
+  void setFrameHeight(size_t h) const { frameHeight = h; }
```

Comment:
The `out_` is const in `paddle/parameter/Argument.h` and these two functions are called by `out_` in `conv_projection`.